packages/kyt-runtime/babel.js

```diff
@@ -0,0 +1,122 @@
+module.exports = function kytRuntimeBabel({ types: t }) {
```

Comment:
This thing could probably use some intro comments on how it is used for `dynamic`, etc. Or we can follow-up with docs, but this might need comments for devs not users